modules/ROOT/pages/metrics/manually-adding-services-to-metrics.adoc

```diff
@@ -47,22 +49,22 @@ documentation on scrape_config] for more details.
 
 === Via the OpenShift Console
 
-1.  Select your project in the `My Projects` sidebar
+.  Select your project in the `My Projects` sidebar
```

Comment:
Should `My Projects` be /* My Projects /* ?

Reply:
whole topic needs review